imputeme/v1.0.4/Dockerfile

```diff
@@ -0,0 +1,225 @@
+################## BASE IMAGE ######################
+
+FROM rocker/shiny:4.0.1
+
+
+################## METADATA ######################
+
+LABEL base_image="https://hub.docker.com/repository/docker/lassefolkersen/impute-me"
+LABEL version="v1.0.4"
```

Comment:
should be version 1

version matches the Dockerfile version, not the upstream version